Monika After Story/game/script-topics.rpy

```diff
@@ -6288,6 +6288,116 @@ label monika_yellowwp:
             m 1a "It'll definitely be an interesting read for you."
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_driving",category=['monika'],prompt="Can you drive?",random=True))
+
+label monika_driving:
+    m 1d "Hm? Can I drive?"
+    m 1c "I never really thought about getting a driver's licence."
+    m 3a "Public transportation is enough for me usually."
+    m 3j "Walking or cycling can be really nice sometimes too!"
+    m 1a "I guess you could say I never really needed to be able to drive."
+    m 1o "I'm not sure I would even have had time to learn to drive with school and all the activities I did anyway."
+    m 1b "What about you, [player]?"
+    m 1a "Can you drive at all?"
+    menu:
+        "Yes.":
+            m 1j "Oh, really?"
+            m "That's great!"
+            m 1k "Gosh, you're amazing, you know that?"
+            m "I can just think of the things we could do together, ehehe~"
+            m 1e "Driving can also be dangerous too though, but if you can drive, you probably already know that."
+            m "Anything could happen at any time."
+            m 3e "No matter how prepared you are, accidents could happen to anyone."
+            m 2l "I mean, I know you're smart but I still worry about you sometimes."
+            m 2e "I just want you to come back to me safe and sound is all."
+            m 1e "I hope you've never had to experience that, [player], have you?"
+            menu:
+                "I've been in an accident before.":
+                    m 2f "Oh..."
+                    m 2o "Sorry to bring that up, [player]..." 
+                    m 2p "I just..."
+                    m 2f "I hope it wasn't too bad."
+                    m 2n "I mean, here you are with me so it must have been alright."
+                    m 2q "..."
+                    m 2e "I'm... glad you survived, [player]..."
```

Comment:
fun fact, what if the player had the accident before even playing ddlc and died, monika wouldnt havent met him thus never be really saved by the player

Reply:
If the player died before meeting Monika, then yeah I guess Monika wouldn't have been saved by them but if they're able to read the dialogue, I think there's a decent chance they lived and got the chance to meet Monika. 

I mean, I got into a car accident before DDLC was released hence why I got the idea to write this topic and last time I checked, I was still alive.